src/remote/action.go

```diff
@@ -27,27 +27,27 @@ import (
 func (c *Client) uploadAction(target *core.BuildTarget, isTest, isRun bool) (*pb.Command, *pb.Digest, error) {
 	var command *pb.Command
 	var digest *pb.Digest
-	err := c.uploadBlobs(func(ch chan<- *chunker.Chunker) error {
+	err := c.uploadBlobs(func(ch chan<- *uploadinfo.Entry) error {
```

Comment:
I don't understand this new nomenclature but whatever

Reply:
I think an `uploadinfo.Entry` is a more abstract thing you're going to upload, a `Chunker` breaks one of those into pieces to push it.

In the new version the former is more relevant here (you can't ask the Chunker for its digest, for example)